products/anolis8/profiles/standard.profile

```diff
@@ -0,0 +1,728 @@
+documentation_complete: true
+
+title: 'Standard System Security Profile for Anolis OS 8'
+
+description: |-
+    This profile contains rules to ensure standard security baseline
+    of a Anolis OS 8 system.
```

Comment:
I wonder on what is this profile based on.

Reply:
Thanks for your reply. Anolis OS is an open source operating system community in China. The community developers combined their experience in operating system compliance to create the Anolis OS 8 Security Benchmark (https://gitee.com/anolis/security-benchmark), a profile based on the Anolis OS 8 Security Benchmarks.